assets/helpers/payPalExpressCheckout/__tests__/payPalExpressReducersTest.js

```diff
@@ -11,7 +11,7 @@ describe('PayPal Reducer Tests', () => {
 
   it('should return the initial state', () => {
 
-    expect(reducer(undefined, {})).toMatchSnapshot();
+    expect(reducer(13, 'GBP')(undefined, {})).toMatchSnapshot();
```

Comment:
By calling with `undefined` I think you override the state that you are defining. Is this what you want to do?